src/containers/MasterWindow.js

```diff
@@ -112,7 +112,7 @@ class MasterWindow extends Component {
                         isAdvanced={modal.isAdvanced}
                         viewId={null}
                         closeCallback={(isNew) => this.closeModalCallback(
```

Comment:
And in that case, what is the reason to wrap method execution in another function and passing the same, one and only parameter?